<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndAntennaDesigner</class>
 <widget class="QDialog" name="wndAntennaDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Antenna Farfield viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="Antennas">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Available Antennas</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Antenna Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnSave">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset theme="document-save">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="btnImportFF">
        <property name="text">
         <string>Import FFIO..</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Field Component</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select value</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QPushButton" name="btnExportWs">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Export FF to Workspace</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QComboBox" name="cbFieldSelect">
        <item>
         <property name="text">
          <string>Etheta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ephi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gain</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Antenna Input Signal</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="lblFreq">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="cbSelectVariable"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnEvalRadiatedField">
        <property name="text">
         <string>Eval Field</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QSlider" name="cbFreqSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QComboBox" name="cbComplexField">
        <item>
         <property name="text">
          <string>Real</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Imag</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magnitude</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phase</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mag(dB)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lnAntennaName"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btnLoad">
     <property name="text">
      <string>Load</string>
     </property>
     <property name="icon">
      <iconset theme="document-open">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Bring to Project</string>
     </property>
     <property name="icon">
      <iconset theme="document-send">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QComboBox" name="cbAvailableAntennas">
     <item>
      <property name="text">
       <string>[Create new antenna...]</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2" colspan="4">
    <widget class="QVTKOpenGLNativeWidget" name="plot" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
